src/components/create-todo/index.js

```diff
@@ -21,14 +22,13 @@ export default function CreateTodo({ addTodo }) {
     setTodo((prev) => {
       return {
         ...prev,
-
         [name]: value,
       };
     });
   };
 
   return (
-    <form onSubmit={handelClick}>
+    <form onSubmit={handelSubmit}>
       <input
         placeholder="type your name"
         onChange={handelChange}
```

Comment:
I think that the components of your application can be organized in a good way. Instead of creating a new directory for each component and writing the code in it's own `index.js` file, you can create a file directly with the component name in `component` directory.
Example: Copy all the code from `create-todo` directory and create a new file in `components` directory named `CreateToDo.js`. 
This will ensure clarity in your code and it will be easy for your team to understand your code better 😉